src/styles/card.scss

```diff
@@ -0,0 +1,113 @@
+@import './variables';
+@import './stars';
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: $font-family;
+  background-color: $white;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background-color: $white;
+  padding: 32px 16px 16px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-family: Roboto;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+    color: #060b35;
+  }
+
+  &__code {
+    color: #616070;
+    font-family: Roboto;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin-bottom: 16px;
+  }
+
+  &__reviews {
+    color: #060B35;
+    display: flex;
+    justify-content: space-between;
+  }
+  &__reviews-text {
+    margin: 0;
+    font-size: 10px;
+    font-weight: 400;
+    color: #060b35;
+    align-items: center;
+  }
+
+  &__price-block {
+    margin-top: 24px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  &__price {
+    font-family: Roboto;
```

Comment:
The font-family should be taken from the variables file for consistency. This helps to maintain the code and change the font in one place if needed.